examples/nodejs/aws/eventbridge/webapp/src/App.tsx

```diff
@@ -82,93 +89,181 @@ const App = () => {
+              <span className="ml-2 font-semibold">Momento Topic</span>
             </div>
-            <ArrowLeft />
+            <ArrowLeft/>
             <div className="flex items-center bg-white p-4 shadow-md rounded-lg">
-              <img src={DynamoDbLogo} alt="DynamoDB Logo" className="w-12 h-12" />
+              <img src={DynamoDbLogo} alt="DynamoDB Logo" className="w-12 h-12"/>
               <span className="ml-2 font-semibold">DynamoDB</span>
             </div>
-            <ArrowRight />
+            <ArrowRight/>
             <div className="flex items-center bg-white p-4 shadow-md rounded-lg">
-              <img src={MomentoTopicLogo} alt="Momento Topic Logo" className="w-12 h-12" />
-              <span className="ml-2 font-semibold">Momento Topic</span>
+              <img src={MomentoCacheLogo} alt="Momento Cache Logo" className="w-12 h-12"/>
+              <span className="ml-2 font-semibold">Momento Cache</span>
             </div>
-          </div>
-
-          <div className="flex flex-row justify-evenly mt-2 w-full">
-            <ArrowDown />
-            <ArrowDown />
+            <CornerRightDown />
           </div>
         </div>
 
         <div className="flex flex-col md:flex-row space-y-2 md:space-y-0 md:space-x-4 p-2 mt-2">
-          <div className="bg-white p-4 rounded-lg shadow-md flex-1">
-            <h1 className="font-bold text-teal-700 mb-2">Get Item From Cache</h1>
-            <DescribeCache location={location} />
-          </div>
-          <div className="bg-white p-4 rounded-lg shadow-md flex-1">
-            <h1 className="font-bold text-teal-700 mb-2">Published Items</h1>
+          <div className="bg-white p-4 rounded-lg shadow-md flex-1 space-y-3">
+            <div className={"flex flex-row space-x-2 text-sm items-center"}>
+              <h1 className="font-bold text-teal-700">Published Messages: </h1>
+              <div className="flex rounded-lg">
+                <h2 className="font-semibold">Cache Name: <span className="text-gray-600 ml-2">{cacheName}</span></h2>
+                <span className={"mr-2"}>,</span>
+                <h2 className="font-semibold">Topic Name: <span className="text-gray-600 ml-2">{topicName}</span></h2>
+              </div>
+              <div className="flex flex-row space-x-2">
+                {isSubscribedToTopic ? (
+                  <Tippy content={"Subscribed"} placement={"top"} trigger={"mouseenter"}>
+                    <div><CheckCircle/></div>
+                  </Tippy>
+                ) : (
+                  <Tippy content={"Unsubscribed"} placement={"top"}>
+                    <div><BanCircle/></div>
+                  </Tippy>
+                )}
+              </div>
+            </div>
+            <div className="bg-gray-100 p-2 rounded-lg text-sm">
+                <pre>
+                  <code className={"whitespace-pre-wrap"}>
+                    {`topicClient.subscribe(${cacheName}, ${topicName}, {onItem: onItemCallback})`}
```

Comment:
you need to put quotes around the cache/topic name, here and in the GetItem one.